docs/addons/unleash.md

```diff
@@ -3,13 +3,29 @@
 > Unleash is a feature toggle system, that gives you a great overview of all feature toggles across all your applications and services. It comes with official client implementations for Java, Node.js, Go, Ruby, Python and .Net.
 
 We use [Unleash](https://docs.getunleash.io/) for feature toggling in NAIS and NAV.
-Get started at (unleash.nais.io)[https://unleash.nais.io/].
+Get started at [unleash.nais.io](https://unleash.nais.io/), and you can look at tips for your [feature toggles available here](https://www.unleash-hosted.com/articles/what-are-the-feature-toggle-best-practices).
 
-We only have one supported environment, but if you want to test it out you can use [unleash.dev.intern.nav.no](https://unleash.dev.intern.nav.no/).
+We've only got _one_  supported environment - but if you want to test it out you can use [(https://)unleash.dev.intern.nav.no](https://unleash.dev.intern.nav.no/).
```

Comment:
Hvorfor https i parentes? Er det ikke bedre å enten skrive full url i lenkteteksten, eller ikke?

Reply:
Jeg tolket hvordan det stod før som "dette er det man pleier å copy-paste inn i koden", og der aner jeg at det er helt i været om man skal spesifisere protokoll eller ikke. Garantert rammeverk som snakker med Unleash som legger på protokoll for deg. 

Det var resonnementet til parantesen.